packages/ketcher-react/package.json

```diff
@@ -70,7 +70,8 @@
     "subscription": "^3.0.0",
     "url-search-params-polyfill": "^8.1.0",
     "w3c-keyname": "^2.2.4",
-    "whatwg-fetch": "^3.4.1"
+    "whatwg-fetch": "^3.4.1",
+    "ketcher-core": "^1.0.0"
```

Comment:
I guess it should be sorted by alpabet there

Reply:
done